common/types/loadbalancer.go

```diff
@@ -57,17 +83,30 @@ type RevNATMap map[ServiceID]L3n4Addr
 type LoadBalancer struct {
 	BPFMapMU  sync.RWMutex
 	SVCMap    SVCMap
+	SVCMapID  SVCMapID
```

Comment:
With this new MAP it should be better to have a function that atomically adds and removes a service for both SVCMap and SVCMapID, no?